File: RefugeUA.Entities/VolunteerGroup.cs
using RefugeUA.Entities.Abstracts;
using RefugeUA.Entities.Interfaces;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RefugeUA.Entities
{
    public class VolunteerGroup : Entity
    {
        public VolunteerGroup()
        {
            this.CreatedAt = DateTime.Now;
        }

        public string Title { get; set; } = default!;

        public string DescriptionContent { get; set; } = default!;

        public DateTime CreatedAt { get; set; }

        public ICollection<IUser> Followers { get; set; } = default!;

        public ICollection<IUser> Admins { get; set; } = default!;

        public ICollection<VolunteerEvent> VolunteerEvents { get; set; } = default!;
    }
}

File: RefugeUA.WebApp/refugeua.webapp.client/src/app/core/models/volunteer-group.ts
import { Entity } from "./entity"
import { User } from "./user"
import { VolunteerEvent } from "./volunteer-event"

export interface VolunteerGroup extends Entity {
    title: string
    descriptionContent: string,
    createdAt: Date,
    followers?: User[],
    admins?: User[],
    followersCount?: number,
    administratorsCount?: number,
    volunteerEvents?: VolunteerEvent[],
    volunteerEventsCount?: number
}